### PR TITLE
Schema fixes

### PR DESCRIPTION
**Pull request by mauerk**
Fixes issue (if applicable) #

## Proposed Changes

  - sequencing protocol type: flag updated to optional
  - biospecimen slot in sample: flag updated to recommended
  - phenotypic feautre: flag updated to recommended
  - added CV for storage slot
  - order of linkage slots updated
  - lane_number deprecated
  
## Checks
  - [ ] Builds locally

### CODE REVIEW

**Review by anandhi-iyappan — DISMISSED** [review]

File: src/schema/submission.yaml [hunk 1]
```diff
@@ -308,6 +311,15 @@ classes:
       cell_barcode_offset:
         required: false
         recommended: true
+      cdna_read:
```
Comment (sbilge) [inline src/schema/submission.yaml:314]: It would be good to separate the slot additions related to single-cell data to another PR?

**Review by sbilge — CHANGES_REQUESTED** [review]
Can you separate the slots added for single-cell data to another PR?

File: src/schema/submission.yaml [hunk 2]
```diff
@@ -165,9 +165,8 @@ classes:
     description: >-
       An condition that is linked to comparable samples.
     slots:
-      - title
```
Comment (sbilge) [inline src/schema/submission.yaml:168]: Shall we unify this through the model? Like either name the other title slots in different classes as "name", or the "name"s as title.

**Review by sbilge — CHANGES_REQUESTED** [review]
Clarification of the difference in using title or name and correcting the corresponding slots  are needed.

**Review by sbilge — APPROVED** [review]

**Pull request merged**